CRM/Civicase/Hook/alterContent/CivicaseSettingsForm.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+class CRM_Civicase_Hook_alterContent_CivicaseSettingsForm {
```

Comment:
This class is not being used presently because it is not referenced in civicase.php file. The block of code in the `civicase_civicrm_alterContent` function is still the one being used. IF we need to use this, we need to remove the code and use this class.

Also the functions in this class does not have documentation

Reply:
Sorry, this class should not have been included. In fact, this and the alter content hook are going to be removed.